docs/source/get_all_venues.rst

```diff
@@ -1,23 +0,0 @@
-Getting All Venues
-==========================
-
-::
-
-    >>> import openreview
-    >>> c = openreview.Client(baseurl='https://api.openreview.net')
-    >>> venues = openreview.tools.get_all_venues(c)
```

Comment:
there is another way of doing this, client.get_group(id='venues'), I'm not sure if the documentation has this.

Reply:
I did copy this into the docs, I can change it to get_group(id='venues')

Reply:
venues contains all the venue ids. The host group only contain the root groups.